src/diffpy/utils/scattering_objects/diffraction_objects.py

```diff
@@ -17,7 +17,7 @@
 )
 
 
-class Diffraction_object:
+class DiffractionObject:
```

Comment:
we want this change, but we want to do it as a deprecation.  Please see the discussion on #163